src/sections/upload-dataset-files/FileUploader.module.scss

```diff
@@ -9,10 +9,36 @@
     grid-template-columns: repeat(4, auto);
 }
 
+.uploaded_files {
+    display: grid;
+    grid-template-columns: 4rem auto auto auto 3.5rem;
+}
+
+.selected_file>div {
```

Comment:
Please don't use !important .
Also this selector points to another class that you already have declared that is selected_files_cb.
Which also could be renamed to selected_file_checkbox for better understanding.

Reply:
I did rename the _cb to _checkbox, but I could not get rid off the !important. I have this code:
```ts
<div className={cn(styles.file, { [styles.selected_file]: selected.has(file) })} key={file.key}>
```

Followed by the css:
```css
.selected_file>div {
    margin-bottom: 0.5rem;
    padding: 0.5rem;
    background-color: #ffc !important;
}

.file:nth-of-type(even)>div {
    margin-bottom: 0.5rem;
    padding: 0.5rem;
    background-color: #f8f9fc;
}

.file:nth-of-type(odd)>div {
    margin-bottom: 0.5rem;
    padding: 0.5rem;
    background-color: white;
}
```

I expect the background colors to look like this:
![image](https://github.com/user-attachments/assets/115a2da2-186b-4e66-bb0f-860b35552c7e)

The selected background does not work when `!important` is not added. I sit because of the checkbox? Not sure how to fix it.

Reply:
I added a task in the follow-up issue so we do not forget to fix it later.